chromium_src/third_party/blink/renderer/modules/canvas/canvas2d/base_rendering_context_2d.cc

```diff
@@ -8,13 +8,32 @@
 #include "base/notreached.h"
 #include "brave/third_party/blink/renderer/core/farbling/brave_session_cache.h"
 #include "third_party/blink/renderer/platform/graphics/image_data_buffer.h"
+#include "third_party/blink/renderer/platform/weborigin/kurl.h"
 
-#define BRAVE_GET_IMAGE_DATA                                                  \
-  if (ExecutionContext* context = ExecutionContext::From(script_state)) {     \
-    SkPixmap image_data_pixmap = image_data->GetSkPixmap();                   \
-    brave::BraveSessionCache::From(*context).PerturbPixels(                   \
-        static_cast<const unsigned char*>(image_data_pixmap.writable_addr()), \
-        image_data_pixmap.computeByteSize());                                 \
+namespace blink {
```

Comment:
this isn't really necessary here, you can just have the anonymous namespace by itsefl

Reply:
fixed